arangod/Graph/PathManagement/PathValidator.h

```diff
@@ -112,6 +114,12 @@ class PathValidator {
       _uniqueEdges;
 
   PathValidatorOptions _options;
+  bool _isSatelliteLeader;
```

Comment:
Please make this part of the PathValidatorOptions. this is constant per run.

Reply:
Done.